test/ngspice_high_level.jl

```diff
@@ -0,0 +1,14 @@
+using NgSpice
+using Test
+NgSpice.init()
+nl = """
+* Diode test circuit
+V1 inp 0 DC=1 AC=1 SIN(1 1 1000)
+* Connected in forward bias:
+D1 inp 0 dmod
+.model dmod d cjo=0 m=1 rs=5000 area=1 pj=0 cjp=0 mj=0 mjsw=0 pb=1 php=1 n=2 bv=0 ibv=0 ik=0 ikr=0 is=1.0e-9 tt=0 xvtfwd=3
```

Comment:
`xvtfwd` isn't recognized as a parameter. Should a library be included?

 This ckt didn't simulate when passed as a string or a file (and subsequently `run`) or on the original Ngspice.

Reply:
you can remove it.